include/cloog/clast.h

```diff
@@ -158,6 +164,10 @@ struct clast_for *new_clast_for(CloogDomain *domain, const char *it,
                                 struct clast_expr *LB, struct clast_expr *UB,
                                 CloogStride *stride);
 struct clast_guard *new_clast_guard(int n);
+struct clast_stmt *clast_substitutions_dup(struct clast_stmt *in_sub);
```

Comment:
where is this defined?

Reply:
The calls to this function have been replaced by `clast_ast_dup` making the function obsolete. The declaration will be removed in the next commit.

Reply:
Please squash all commits into one and force push so that all of this appears as one commit.